src/script/components/TitleBar/index.tsx

```diff
@@ -17,395 +17,4 @@
-                >
-                  <Icon.Camera />
-                </button>
-              )}
-
-              <button
-                type="button"
-                className="conversation-title-bar-icon"
-                title={t('tooltipConversationCall')}
-                aria-label={t('tooltipConversationCall')}
-                onClick={() => callActions.startAudio(conversation)}
-                data-uie-name="do-call"
-              >
-                <Icon.Pickup />
-              </button>
-            </div>
-          )}
-
-          {isScaledDown ? (
-            <>
-              <IconButton
-                className="icon-search"
-                css={{marginBottom: 0}}
-                title={t('tooltipConversationSearch')}
-                aria-label={t('tooltipConversationSearch')}
-                onClick={onClickCollectionButton}
-                data-uie-name="do-collections"
-              >
-                <span className="visually-hidden">{t('tooltipConversationSearch')}</span>
-              </IconButton>
-
-              <IconButton
-                title={t('tooltipConversationCall')}
-                aria-label={t('tooltipConversationCall')}
-                css={{marginBottom: 0}}
-                onClick={() => callActions.startAudio(conversation)}
-                data-uie-name="do-call"
-              >
-                <Icon.Pickup />
-              </IconButton>
-            </>
-          ) : (
-            <button
-              type="button"
-              title={t('tooltipConversationInfo')}
-              aria-label={t('tooltipConversationInfo')}
-              onClick={onClickDetails}
-              className={cx('conversation-title-bar-icon', {active: isRightPanelOpen})}
-              data-uie-name="do-open-info"
-            >
-              <Icon.Info />
-            </button>
-          )}
-        </li>
-
-        {badgeLabelCopy && (
-          <li
-            className="conversation-title-bar-indication-badge"
-            data-uie-name="status-indication-badge"
-            dangerouslySetInnerHTML={{__html: badgeLabelCopy}}
-          />
-        )}
-      </ul>
-    </StyledApp>
-  );
-};
-
-export function generateWarningBadgeKey({
-  hasFederated,
-  hasExternal,
-  hasGuest,
-  hasService,
-}: {
-  hasExternal?: boolean;
-  hasFederated?: boolean;
-  hasGuest?: boolean;
-  hasService?: boolean;
-}): StringIdentifer {
-  const baseKey = 'guestRoomConversationBadge';
-  const extras = [];
-  if (hasGuest && !hasExternal && !hasService && !hasFederated) {
-    return baseKey;
-  }
-  if (hasFederated) {
-    extras.push('Federated');
-  }
-  if (hasExternal) {
-    extras.push('External');
-  }
-  if (hasGuest) {
-    extras.push('Guest');
-  }
-  if (hasService) {
-    extras.push('Service');
-  }
-  if (!extras.length) {
-    return '';
-  }
-  return `${baseKey}${extras.join('And')}` as StringIdentifer;
-}
-
-export default TitleBar;
+export {default} from './TitleBar';
```

Comment:
Let's avoid exporting `default` from now on (we now have an eslint rule to avoid this, thanks to @aweiss-dev :))
Let's rather `export * from './TitleBar`